### PR TITLE
feat: link to cmc token

### PR DESCRIPTION
**Pull request by zhouhanseng**
Close https://github.com/DimensionDev/Maskbook/issues/1726

### CODE REVIEW

File: src/plugins/Trader/UI/trending/PriceChart.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ const useStyles = makeStyles((theme: Theme) => {
     return createStyles({
         root: {
             position: 'relative',
+            cursor: (props: PriceChartProps) => (props.stats.length ? 'pointer' : 'default'),
```
Comment (septs) [inline src/plugins/Trader/UI/trending/PriceChart.tsx:21]:
```suggestion
            cursor: ({ stats }: PriceChartProps) => (stats.length ? 'pointer' : 'default'),
```

File: src/plugins/Trader/UI/trending/TrendingView.tsx [hunk 2]
```diff
@@ -275,6 +276,7 @@ export function TrendingView(props: TrendingViewProps) {
 
     const { coin, market, tickers } = trending
     const canSwap = trending.coin.eth_address || trending.coin.symbol.toLowerCase() === 'eth'
+    const cmcCurrencyURL = CMC_CURRENCIES_URL + '/' + coin.name.toLocaleLowerCase()
```
Comment (septs) [inline src/plugins/Trader/UI/trending/TrendingView.tsx:279]:
```suggestion
    const cmcCurrencyURL = CMC_CURRENCIES_URL + '/' + coin.name.toLocaleLowerCase()
```

template string

File: .github/ISSUE_TEMPLATE/demand.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ name: Demand
 about: PRD for new demand
 title: '[Demand] '
 labels: 'Demand: Draft'
-assignees: 
+assignees:
```
Comment (septs) [inline .github/ISSUE_TEMPLATE/demand.md:6]: why change this file?

Comment (zhouhanseng) [reply .github/ISSUE_TEMPLATE/demand.md:6]: prettier

File: src/plugins/Trader/UI/trending/TrendingView.tsx [hunk 4]
```diff
@@ -275,6 +276,7 @@ export function TrendingView(props: TrendingViewProps) {
 
     const { coin, market, tickers } = trending
     const canSwap = trending.coin.eth_address || trending.coin.symbol.toLowerCase() === 'eth'
+    const cmcCurrencyURL = `${CMC_CURRENCIES_URL}/${coin.name.toLocaleLowerCase()}`
```
Comment (septs) [inline src/plugins/Trader/UI/trending/TrendingView.tsx:279]: not require use `toLocale*`

Comment (septs) [reply src/plugins/Trader/UI/trending/TrendingView.tsx:279]: see https://mdn.io/toLocaleLowerCase
see https://mdn.io/toLowerCase

**Review by septs — APPROVED** [review]

**Pull request merged**